### PR TITLE
Documentation enhancements based on #159

### PR DESCRIPTION
**Pull request by romanpeters**
Added step-by-step instructions for setting up a new development or production server.
Inspired by issue #159

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,24 +128,52 @@ should be used that bootstraps the server. It installs Ansible's dependencies,
...
+### Step-by-step guide
+These are the steps to follow to set up a new development or production server.
+Some of the steps require you to specify which of the two you are setting up.
+
+If you want to migrate from an existing server, a few additional tasks should be
+performed, which are explained in detail in [this guide][deployment-new-production].
+
+##### On Digital Ocean:
+1. Create a droplet named either `dev.svsticky.nl` (staging) or `svsticky.nl` (production).
+1. Assign a floating IP to the new droplet.
```
Comment (tomwassenberg) [inline README.md]: Can you add the reason? (Because the floating IP's are already in DNS, which avoids DNS cache problems)

File: README.md [hunk 2]
```diff
@@ -149,24 +128,52 @@ should be used that bootstraps the server. It installs Ansible's dependencies,
...
+
+1. Bootstrap the host for either production or staging.
+`$ ./scripts/run-playbook.sh (production|staging) bootstrap-new-host.yml`
+You do not need to enter a SUDO password, but you do need to enter the correct Vault password.
+At the end of the process you will receive a newly generated SUDO password, which you will need in the next step.
+
+1. Run the main playbook for either production or staging.
+`$ ./scripts/run-playbook.sh (production|staging) main.yml`
+
+1. If you are not migrating from a previous version of the server, you will need to run these two commands to enable Koala.
```
Comment (tomwassenberg) [inline README.md]: I think `start.yml` also needs to be run when migrating from a previous server. It just needs to be done *after* you've manually migrated the db.

File: README.md [hunk 3]
```diff
@@ -182,5 +189,8 @@ Godspeed!
   [Ansible Vault]:http://docs.ansible.com/ansible/playbooks_vault.html
   [inventory]:https://docs.ansible.com/ansible/intro_inventory.html
   [slacktee]:https://github.com/course-hero/slacktee
+  [ansible]:https://github.com/ansible/ansible
   [deployment-new-production]:docs/deployment-new-production.md
   [IT Crowd]:mailto:itcrowd@svsticky.nl
+  [deployment-guide]:https://github.com/svsticky/sadserver/blob/master/README.md#setting-up-the-staging-and-production-environment
```
Comment (tomwassenberg) [inline README.md]: If you leave out everything before the anchor, the link will also work on other branches, like this one ;)

Comment (arianvp) [reply README.md]: that is `[deployment-guide](setting-up-the-staging-and-production-environment)` by the way :P

**Pull request merged**